.travis.yml

```diff
@@ -1,9 +1,13 @@
 language: node_js
 node_js: ["0.10"]
+cache:
+  directories:
+    - node_modules
+    - bower
 before_install:
   - export CHROME_BIN=chromium-browser
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
 before_script:
-  - npm install -g bower gulp
```

Comment:
Will removing gulp here make the tests fail if the cache is removed? Looks like we might want to switch the test script to use `./node_modules/.bin/gulp` then instead of the global gulp.